server/mldb_server.cc

```diff
@@ -499,8 +499,8 @@ getPackageDocumentationPath(const Package & package) const
     // by any means guaranteed.
 
     if (package.packageName() == "builtin")
-        return "/doc/builtin/";
-    return "/v1/plugins/" + package.packageName() + "/doc/";
+        return httpBaseUrl + "/doc/builtin/";
+    return httpBaseUrl + "/v1/plugins/" + package.packageName() + "/doc/";
```

Comment:
Since httpBaseUrl is a user input I would use the safer boost::filesystem operator/.